.contributing/the-process.md

```diff
@@ -125,4 +154,24 @@ Notes on the various tools 
 
 -   Google docs are great for creating longer, more complex documents (eg. with diagrams and a TOC)
 
--   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
\ No newline at end of file
+-   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
+
+
+Key
+====
+
+- "IS" : input session
+- "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* < User Flow files < *Section* (CL)"
+- "input task": 
+  - e.g Create a task in Asana with title "IS: *title of Google doc*" and:
+      - Add a link to the google drawing in the task description
```

Comment:
Bugger, now this seems a little confusing to me - because a google doc is not a google drawing and we're using the terms interchangeably.  I don't have a good suggestion off the top of my head - could "*title of Google doc*" just be changed to *title of Google drawing*"?  Happy with other solutions, also happy if others think it's fine.

Reply:
good point! and its not always a google drawing, one of them is a google doc.

how bout "IS:  *title of google file*"?

Reply:
or "IS: *title of google doc/drawing*"

Reply:
SGTM!

Reply:
@nickmeinhold

Reply:
After a chat - I agree with @tamari-gray - google file sounds great!